Apps/Examples/Examples/All Examples/CustomAnchoredSymbolExample.swift

```diff
@@ -0,0 +1,321 @@
+    var label: String
+    var imageName: String
+}
+
+@objc(CustomAnchoredSymbolExample)
+public class CustomAnchoredSymbolExample: UIViewController, ExampleProtocol {
+    static let symbols = "custom_symbols"
+
+    internal var mapView: MapView!
+
+    // Configure a label
+    public lazy var label: UILabel = {
+        let label = UILabel(frame: CGRect.zero)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.backgroundColor = UIColor.systemBlue
+        label.layer.cornerRadius = 12.0
+        label.textColor = UIColor.white
+        label.textAlignment = .center
+        label.font = UIFont.boldSystemFont(ofSize: 24.0)
+        return label
+    }()
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Center the map camera over New York City
+        let centerCoordinate = CLLocationCoordinate2D(latitude: 40.7128, longitude: -74.0060)
+        let options = MapInitOptions(cameraOptions: CameraOptions(center: centerCoordinate,
+                                                                  zoom: 15.0))
+
+        mapView = MapView(frame: view.bounds, mapInitOptions: options)
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        view.addSubview(mapView)
+
+        let tapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector((mapSymbolTap(sender:))))
+        mapView.addGestureRecognizer(tapGestureRecognizer)
+
+        // Allows the delegate to receive information about map events.
+        mapView.mapboxMap.onNext(.mapLoaded) { [weak self] _ in
+            guard let self = self else { return }
+
+            self.updateSymbolImages()
+            let features = self.addFeatures()
+            self.addSymbolLayer(features: features)
+
+            // The below line is used for internal testing purposes only.
+            self.finish()
+        }
+
+        // Add the label on top of the map view controller.
+        addLabel()
+    }
+
+    public func addLabel() {
+        label.text = "Select A Marker"
+        view.addSubview(label)
+
+        NSLayoutConstraint.activate([
+            label.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            label.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            label.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            label.heightAnchor.constraint(equalToConstant: 60.0)
+        ])
+    }
+
+    @objc private func mapSymbolTap(sender: UITapGestureRecognizer) {
+        if sender.state == .recognized {
+            let layers: [String] = [CustomAnchoredSymbolExample.symbols]
+
+            mapView.mapboxMap.queryRenderedFeatures(at: sender.location(in: mapView),
+                                                    options: RenderedQueryOptions(layerIds: layers, filter: nil)) { result in
+                switch result {
+                case .success(let features):
+                    if features.count > 0 {
```

Comment:
```suggestion
                    if !features.isEmpty {
```

See docs on isEmpty for the recommended best practice